tests/inputFields.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { test, expect } from '@playwright/test'
+
+test.describe('Input fields', async () => {
+	test.beforeEach(async ({ page }) => {
+		await page.goto('/')
+		await page.getByText('Pet Types').click()
+		await expect(page.getByRole('heading', { name: 'Pet Types' })).toBeVisible()
+	})
+
+	test('Update pet type', async ({ page }) => {
+		// Test data
+		const originalPetType = 'cat'
+		const newPetType = 'rabbit'
+
+		// Locators - Pet Types table page
+		const firstEditButton = page.getByRole('button', { name: 'Edit' }).first()
+		const firstPetTypeInput = page.locator('[name="pettype_name"]').first()
+		// Locators - Edit Pet Types page
+		const editPetTypeHeader = page.getByRole('heading', { name: 'Edit Pet Type' })
+		const petTypeInput = page.locator('#name')
+		const petTypeUpdateButton = page.getByRole('button', { name: 'Update' })
```

Comment:
You don't need to define all locators in advance. It's more of "Selenium" style, it was a thing when people used unreadable XPath selectors. Here, each locator is very readable. This is especially a case when locator was used just once in the code, for example `editPetTypeHeader` in this example.

Constants should be created when you need to reuse the locators several times.